docker-bake.hcl

```diff
@@ -64,8 +76,8 @@ target "_common" {
     DOCKER_STATIC = DOCKER_STATIC
     DOCKER_LDFLAGS = DOCKER_LDFLAGS
     DOCKER_BUILDTAGS = DOCKER_BUILDTAGS
-    DOCKER_GITCOMMIT = DOCKER_GITCOMMIT
-    VERSION = VERSION
+    DOCKER_GITCOMMIT = GITHUB_SHA != "" ? GITHUB_SHA : DOCKER_GITCOMMIT
+    VERSION = GITHUB_REF != "" ? GITHUB_REF : VERSION
```

Comment:
If feels like the order of preference should be reversed here; `DOCKER_GITCOMMIT` and `VERSION` should (probably?) take precedence if both exist.